### PR TITLE
Moving to 30GeV jet counting

### PR DESCRIPTION
**Pull request by zhenbinwu**
As discussed on Monday, we will move the nJet counting to 30GeV while keeping the b-tagged jet at 20GeV.

### CODE REVIEW

**Comment by zhenbinwu** [comment]
any comments on this PR @wang-hui @pastika @mkilpatr

File: python/modules/Stop0lObjectsProducer.py [hunk 1]
```diff
@@ -229,8 +231,7 @@ def analyze(self, event):
         self.IsoTrack_Stop0l = map(lambda x : self.SelIsotrack(x, met), isotracks)
         self.Tau_MtW	     = map(lambda x : self.CalMtW(x, met), taus)
         self.Jet_Stop0l      = map(self.SelJets, jets)
-        local_BJet_Stop0l    = map(self.SelBtagJets, jets)
-        self.BJet_Stop0l     = [a and b for a, b in zip(self.Jet_Stop0l, local_BJet_Stop0l )]
```
Comment (pastika) [inline python/modules/Stop0lObjectsProducer.py]: What is this change about?

Comment (zhenbinwu) [reply python/modules/Stop0lObjectsProducer.py]: no using the Jet_Stop0l (30GeV cut now). So that Btag is selected with 20GeV cut

**Comment by zhenbinwu** [comment]
Ok, this has been checked by Hui and also approved by Matt. Merging

**Pull request merged**